transform/mattermost-analytics/models/marts/nps/dim_nps_feedback.sql

```diff
@@ -0,0 +1,23 @@
+select distinct nf.server_id as server_id
+       , nf.user_id as user_id
+       , nf.license_id as license_id
+       , nf.feedback as feedback
+       , ns.score as score
+       , nf.server_version as server_version
+       , COALESCE(cc.license_name, sh.license_name, nf.hosting_type) as hosting_type
```

Comment:
could the `license_name` be used as a substitute for `hosting_type`? We have explores where we filter by `hosting_type`, will it affect consistency across?

Reply:
Removed this field, to do the join in Looker and python code. Also converted this to a `fct` model and there is no extra reference to other models. This model is only NPS related data.